.changelog/35873.txt

```diff
@@ -0,0 +1,6 @@
+@@ -0,0 +1,7 @@
```

Comment:
whats this line?

Reply:
Fixed now. The diff line seems to have crept in during merge conflict resolution.